vowpalwabbit/cb_explore.cc

```diff
@@ -375,8 +376,7 @@ base_learner* cb_explore_setup(options_i& options, vw& all)
       data->epsilon_decay = true;
     }
     data->cs = (learner<cb_explore, example>*)(as_singleline(all.cost_sensitive));
-    data->second_cs_label.costs.resize(num_actions);
-    data->second_cs_label.costs.end() = data->second_cs_label.costs.begin() + num_actions;
+    for (uint32_t j = 0; j < num_actions; j++) { data->second_cs_label.costs.push_back(COST_SENSITIVE::wclass{}); }
```

Comment:
make an `actual_resize()` function?

Reply:
I'd like to do that in a follow up PR to avoid this one doing too much.